### DISCUSSION TITLE
How much resources this plugin takes

### DISCUSSION THREAD

**Discussion by cuddleandtea** [WarmUpTill/SceneSwitcher#739] [Q&A]
Hello! I'm started to use this plugin and it's awesome! So many things I can make. But how much does it affect the performance and what is the best option for it to be optimized?

**Comment by WarmUpTill** [comment]
Unless you are using resource intensive conditions / actions (e.g. Video condition's object detection, File condition in combination with huge files) the resource impact should be rather small.

The main control to adjust the performance impact is found on the General tab - the "Check conditions every ..." option.

Let me know if you have any more specific questions! :)

**Comment by nacalai** [comment]
I can add that we're using a ton of macros for our financial news show. Works like a charm. I see no performance impact
